scripts/core/editor3/components/tests/editor3.spec.jsx

```diff
@@ -33,6 +33,51 @@ describe('editor3.component', () => {
             expect(onDragOver(makeEvent(invalidType))).toBe(true);
         });
     });
+
+    it('should not accept dragging when editor is readOnly', () => {
+        const wrapper = shallow(<Editor3 readOnly={true} editorFormat={['picture']} singleLine={false}/>);
+        const {onDragOver} = wrapper.instance();
+        const makeEvent = (t) => ({originalEvent: {dataTransfer: {types: [t]}}});
+
+        [
+            'application/superdesk.item.picture',
+            'application/superdesk.item.graphic',
+            'application/superdesk.item.video',
+            'text/html'
+        ].forEach((validType) => {
+            expect(onDragOver(makeEvent(validType))).toBe(true);
```

Comment:
This is not a valid test because you don't know which return branch resulted in the value `true` and whether the drag was processed or not (both your branches return true in `onDragOver`).